frontend/src/pages/booking/components/BookingForm/BookingForm.tsx

```diff
@@ -62,16 +68,20 @@ function BookingForm({
   const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
 
-    if (!sharingAgreed) {
-      setErrored(true);
-      return;
-    }
+    const newErrors = {
+      checkbox: !sharingAgreed,
+      textarea: !!detailErrorMessage,
+    };
+
+    setErrored(newErrors);
+
+    if (newErrors.checkbox || newErrors.textarea) return;
```

Comment:
p3: {} 로 감싸는 게 적용 안된 것 같아!

```suggestion
    if (newErrors.checkbox || newErrors.textarea){
      return;
    }
```